### ISSUE TITLE
Error in project yaml after updating to 1.2.1

### ISSUE THREAD

**Issue by boboldehampsink** [verbb/navigation#123]
### Description

My site won't run on 1.2.1. It shows the following error:

```
web_1         |     > applying existing project config ... Exception: Undefined index: maxNodes (/app/user/vendor/verbb/navigation/src/services/Navs.php:191)
web_1         | #0 /app/user/vendor/verbb/navigation/src/services/Navs.php(191): yii\base\ErrorHandler->handleError(8, 'Undefined index...', '/app/user/vendo...', 191, Array)
web_1         | #1 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(872): verbb\navigation\services\Navs->handleChangedNav(Object(craft\events\ConfigEvent))
web_1         | #2 [internal function]: craft\services\ProjectConfig->craft\services\{closure}(Object(craft\events\ConfigEvent))
web_1         | #3 /app/user/vendor/yiisoft/yii2/base/Component.php(627): call_user_func(Object(Closure), Object(craft\events\ConfigEvent))
web_1         | #4 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(551): yii\base\Component->trigger('addItem', Object(craft\events\ConfigEvent))
web_1         | #5 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(866): craft\services\ProjectConfig->processConfigChanges('navigation.navs...')
web_1         | #6 [internal function]: craft\services\ProjectConfig->craft\services\{closure}(Object(craft\events\ConfigEvent))
web_1         | #7 /app/user/vendor/yiisoft/yii2/base/Component.php(627): call_user_func(Object(Closure), Object(craft\events\ConfigEvent))
web_1         | #8 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(551): yii\base\Component->trigger('addItem', Object(craft\events\ConfigEvent))
web_1         | #9 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(957): craft\services\ProjectConfig->processConfigChanges('navigation.navs...')
web_1         | #10 /app/user/vendor/craftcms/cms/src/services/ProjectConfig.php(441): craft\services\ProjectConfig->_applyChanges(Array)
web_1         | #11 /app/user/vendor/craftcms/cms/src/migrations/Install.php(1036): craft\services\ProjectConfig->applyYamlChanges()
web_1         | #12 /app/user/vendor/craftcms/cms/src/migrations/Install.php(73): craft\migrations\Install->insertDefaultData()
web_1         | #13 /app/user/vendor/craftcms/cms/src/db/Migration.php(58): craft\migrations\Install->safeUp()
web_1         | #14 /app/user/vendor/craftcms/cms/src/db/MigrationManager.php(239): craft\db\Migration->up(true)
web_1         | #15 /app/user/vendor/craftcms/cms/src/console/controllers/InstallController.php(161): craft\db\MigrationManager->migrateUp(Object(craft\migrations\Install))
web_1         | #16 [internal function]: craft\console\controllers\InstallController->actionCraft()
web_1         | #17 /app/user/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
web_1         | #18 /app/user/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
web_1         | #19 /app/user/vendor/yiisoft/yii2/console/Controller.php(164): yii\base\Controller->runAction('', Array)
web_1         | #20 /app/user/vendor/craftcms/cms/src/console/Controller.php(199): yii\console\Controller->runAction('', Array)
web_1         | #21 /app/user/vendor/yiisoft/yii2/base/Module.php(528): craft\console\Controller->runAction('', Array)
web_1         | #22 /app/user/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('install', Array)
web_1         | #23 /app/user/vendor/craftcms/cms/src/console/Application.php(93): yii\console\Application->runAction('install', Array)
web_1         | #24 /app/user/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('install', Array)
web_1         | #25 /app/user/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
web_1         | #26 /app/user/craft(23): yii\base\Application->run()
web_1         | #27 {main}
web_1         | Exception 'craft\errors\MigrationException' with message 'An error occurred while executing the "craft\migrations\Install migration: Undefined index: maxNodes'
web_1         | 
web_1         | in /app/user/vendor/craftcms/cms/src/db/MigrationManager.php:256
web_1         | 
web_1         | Stack trace:
web_1         | #0 /app/user/vendor/craftcms/cms/src/console/controllers/InstallController.php(161): craft\db\MigrationManager->migrateUp(Object(craft\migrations\Install))
web_1         | #1 [internal function]: craft\console\controllers\InstallController->actionCraft()
web_1         | #2 /app/user/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
web_1         | #3 /app/user/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
web_1         | #4 /app/user/vendor/yiisoft/yii2/console/Controller.php(164): yii\base\Controller->runAction('', Array)
web_1         | #5 /app/user/vendor/craftcms/cms/src/console/Controller.php(199): yii\console\Controller->runAction('', Array)
web_1         | #6 /app/user/vendor/yiisoft/yii2/base/Module.php(528): craft\console\Controller->runAction('', Array)
web_1         | #7 /app/user/vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction('install', Array)
web_1         | #8 /app/user/vendor/craftcms/cms/src/console/Application.php(93): yii\console\Application->runAction('install', Array)
web_1         | #9 /app/user/vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction('install', Array)
web_1         | #10 /app/user/vendor/yiisoft/yii2/base/Application.php(386): yii\console\Application->handleRequest(Object(craft\console\Request))
web_1         | #11 /app/user/craft(23): yii\base\Application->run()
web_1         | #12 {main}
```

### Steps to reproduce

1. Install 1.2.1 and save to project.yaml
2. Install new site with this project.yaml

### Additional info

- Plugin version: 1.2.1
- Craft version: 3.3.19

**Comment by engram-design** [comment]
What were you updating from (if you know)? Can you check if the migration `m200108_200000_add_max_nodes` has run?

I _think_ this should be fixed in [1.2.2](https://github.com/verbb/navigation/releases/tag/1.2.2)

**Comment by boboldehampsink** [comment]
Works now, thanks
